src/module/apps/compendium-browser/tabs/equipment.ts

```diff
@@ -96,6 +101,15 @@ export class CompendiumBrowserEquipmentTab extends CompendiumBrowserTab {
                         itemData.system.source.value = sluggify(source);
                     }
 
+                    // Infer magical trait from runes
+                    const traits = itemData.system.traits.value ?? [];
+                    if (
+                        (itemData.type === "armor" && itemData.system.potencyRune.value) ||
+                        (itemData.type === "weapon" && itemData.system.strikingRune.value)
```

Comment:
This should probably be looking for a potency rune as well.

Reply:
Updated